modules/articles/client/views/view-article.client.view.html

```diff
@@ -1,4 +1,4 @@
-<section>
+<section ng-init="addArticleTitle()">
```

Comment:
I'd rather see a `init()` method added to the controller, and then have it called initially in the controller. I think we should avoid `ng-init` as some views might be re-used.